installing/installing_gcp/installing-gcp-three-node.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="installing-gcp-three-node"]
+= Installing a three-node cluster on GCP
+include::_attributes/common-attributes.adoc[]
+:context: installing-gcp-three-node
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a three-node cluster on Google Cloud Platform (GCP). A three-node cluster consists of three control plane machines, which also act as compute machines. This type of cluster provides a smaller, more resource efficient cluster, for cluster administrators and developers to use for testing, development, and production.
+
+You can install a three-node cluster using either installer-provisioned or user-provisioned infrastructure.
+
+include::modules/installation-three-node-cluster-cloud-provider.adoc[leveloffset=+1]
```

Comment:
> In this doc, the `Procedure` section should have another step of `create manifests`...

@jianli-wei I do not believe a new step 2 is required. This might cause confusion as the step you are proposing is for a UPI installation only. The goal of the procedure is to effectively let users know that configuring a three-node cluster (IPI or UPI) requires you to set `compute.replicas` to `0` and then cross references ( see Next steps section) the procedures for the standard IPI and UPI installations. Each of these procedures, where required, provide guidance on what to do/not to do if you are configuring a three-node cluster. See the docs preview links in comment zero.

So for example, Installing a cluster on user-provisioned infrastructure in GCP by using Deployment Manager templates > [Creating the Kubernetes manifest and Ignition config files](https://56842--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-user-infra.html#installation-user-infra-generate-k8s-manifest-ignition_installing-gcp-user-infra) instructs users to skip the step that has them changing `mastersSchedulable` to false.

Reply:
@mjpytlak My concerns is about "_If you install a cluster on infrastructure that the installation program provisions, you can complete this verification step **after the cluster is deployed**._", because I don't think the mentioned YAML file would still exist after the cluster is deployed. Thanks!